hack/k8s-in-k8s/generate_env.sh

```diff
@@ -68,6 +68,10 @@ function GetKubernetesCaPath() {
     kubectl get cm kubelet-config -nkube-system -oyaml  | awk '/clientCAFile:/{print $2}'
 }
 
+function GetKubeDnsClusterIP() {
+    kubectl get svc -nkube-system kube-dns  -o jsonpath='{.spec.clusterIP}'
```

Comment:
kubectl may be in other path such as ”/home/deploy/kubectl“ in some env

Reply:
This script is used to assist in generating environment variables, and its results need to be adjusted according to specific circumstances.